### ISSUE TITLE
Line number and + to group functions

### ISSUE THREAD

**Issue by Am4dEuZ** [eried/Arduino#1]
Hi,
great work, very useful. This is only few suggests.
- Add line number to code window (i know that is reported bottom, but near code line if better)
- Add the [+] to group the function (like notepad++ or many other progs). For great projects (i work on mega) i have many lines. Grouping I could work better.

Bye

**Comment by eried** [comment]
I am trying  to add the line numbers but I am not sure about the [+] function (line numbers were requested >4 times by different people but I code a lot in Visual Studio and never really used the collapsed branches for anything rather than big imported code, and in Arduino you can just create a tab). I usually keep more updated this thread: http://forum.arduino.cc/index.php?topic=118440.0

**Comment by Sembazuru** [comment]
I recently installed 1.5.6r2 (just to see what, if anything, was different from 1.0.5) and noticed that real line numbers were added. You may be able to see what they did and implement it. (Or wait and hope that the next version that you apply ERW to already has line numbers...)

**Comment by eried** [comment]
Yes, I am gonna use 1.5.6r2 as a new base because I think they have added lot of useful stuff. I may need to add little few missing enhancements now
